wiki/Reporting_Bad_Behaviour/en.md

```diff
@@ -1,133 +1,112 @@
- \_\_NOTOC\_\_ \_\_NOEDITSECTION\_\_
+# Reporting Bad Behavior
 
-<div style="margin:0 auto;width:calc(100% - 25px);border-radius:10px;border:1px solid #ddd;background-color:white;box-shadow:0 0 5px #ddd;-webkit-
-box-shadow:0 0 5px #ddd;-moz-box-shadow:0 0 5px #ddd;-o-box-shadow:0 0 5px #ddd;">
-<div class="plainlinks" style="padding:20px;">
 What is the report system?
 --------------------------
 
 The report system is an important feature to inform the moderators about misbehaving user(s) and for them to take action.
 
 It is not only limited to in game chat but also in other places such as the forum and wiki itself.
 
-------------------------------------------------------------------------
-
 How to report someone?
 ----------------------
 
-You may report at many places, but let's start at the chat.
+### Using the !report command ingame
 
-### Using the !report command
+Usage:
+`!report username [reason]`
 
-`Format: !report `<username>` `<reason>
+Parameters in **<** and **>** are optional.
 
-Take note that <b>&lt;</b> and <b>&gt;</b> are not necessary.
+If the user has a space at their username, it must be  replaced with an underscore ``_`` because the osu! ingame chat is based on [IRC](IRC wiki/Internet_Relay_Chat).
```

Comment:
`If the user has a space at their username, it must be  replaced with an underscore ``_`` because the osu! ingame chat is based on [IRC](/wiki/Internet_Relay_Chat).`